src/App.tsx

```diff
@@ -17,9 +26,12 @@ function App() {
         )
     }
     if (team?.token !== "admin token") {
-        return (<><TrackPage team={team}></TrackPage></>)
+        return (<TrackPage team={team} removeTeam={storage.removeTeam} setTeam={setTeam}></TrackPage> )
```

Comment:
You can simplify it like this, because you haven't any children:
```
<TrackPage team={team} removeTeam={storage.removeTeam} setTeam={setTeam} />
```